docs/reference/default-theme-config.md

```diff
@@ -214,7 +214,9 @@ export default {
         icon: {
           svg: '<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Dribbble</title><path d="M12...6.38z"/></svg>'
         },
-        link: '...'
+        link: '...',
+        // You can include a custom label for accessibility.
+        label: 'cool link'
```

Comment:
Maybe this should be named `aria-label`? `label` sounded to me like if it were to add actual text after the icon 👀